lib/src/configuration.dart

```diff
@@ -57,6 +57,9 @@ typedef MigrationCallback = void Function(Migration migration, int oldSchemaVers
 /// Configuration used to create a [Realm] instance
 /// {@category Configuration}
 abstract class Configuration implements Finalizable {
+
+  static const int encryptionKeySize = 64;
```

Comment:
I don't think this needs to be public.

Reply:
I use it inside _RealmCore and and in the tests also.

Reply:
Sure, but I don't think it justifies adding an extra public member.

Reply:
I will think where I can move it.